src/sentry/incidents/models.py

```diff
@@ -539,8 +539,11 @@ class Type(Enum):
         PAGERDUTY = 1
         SLACK = 2
         MSTEAMS = 3
+        SENTRY_APP = 4
 
-    INTEGRATION_TYPES = frozenset((Type.PAGERDUTY.value, Type.SLACK.value, Type.MSTEAMS.value))
+    INTEGRATION_TYPES = frozenset(
```

Comment:
I thought I had added a comment before about this, but I don't think we need (or want) to add `SENTRY_APP` to this. From what I can tell this is used in logic when saving and updating alert rules for first party integrations but sentry apps will behave a bit differently